packages/server/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/big.js": "^4.0.4",
     "@types/url-join": "^0.8.2",
     "big.js": "^5.0.3",
+    "crypto-random-string": "^1.0.0",
```

Comment:
This package appears to be all of one line, can we just use said line https://github.com/sindresorhus/crypto-random-string/blob/master/index.js#L9 and avoid yet another external dep?

Tho I am confused about how node.js style `require('crypto')` is working for us, guessing there's some browserify magic happening?